pf4j/src/main/java/org/pf4j/processor/ExtensionAnnotationProcessor.java

```diff
@@ -98,7 +101,29 @@ public boolean process(Set<? extends TypeElement> annotations, RoundEnvironment
 
             TypeElement extensionElement = (TypeElement) element;
 //            Extension annotation = element.getAnnotation(Extension.class);
-            List<TypeElement> extensionPointElements = findExtensionPoints(extensionElement);
+            List<TypeElement> extensionPointElements;
```

Comment:
Try to create a nice generic static method with parameter(s) in Move this method in https://github.com/pf4j/pf4j/blob/master/pf4j/src/main/java/org/pf4j/util/ClassUtils.java and use that method here

Reply:
I'm not sure, if I understand your change request correctly. Does PF4J or some plugin developer need the detection of extension points as global (final & static) method in `ClassUtils`?

I'm assuming, that only the `ExtensionAnnotationProcessor` would ever need this mechanism. It seems more logic to me to put this modification into the private `findExtensionPoints` method, where the automatic detection is already present. I've [updated my pull request](https://github.com/pf4j/pf4j/pull/265/commits/3d929a3d8831ba2acccd31182f9308b8763254d3) to illustrate, how I would approach this.

But if you prefer to have some (or all?) parts of the extension point detection as public method in `ClassUtils`, I will make the necessary changes of course.